src/DataLakeStore/DataLakeStore/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.3.1
+* Added code to prevent delete of destination if destination is parent of path to be moved
+
```

Comment:
No need to set the version manually. Just add the new changelog line next to `## Upcoming Release`

```suggestion
## Upcoming Release
* Added code to prevent delete of destination if destination is parent of path to be moved

```